quickwit/quickwit-cli/src/service.rs

```diff
@@ -80,7 +81,12 @@ impl RunCliCommand {
         quickwit_telemetry::send_telemetry_event(telemetry_event).await;
         // TODO move in serve quickwit?
         start_actor_runtimes(&config.enabled_services)?;
-        serve_quickwit(config).await?;
+        let _ = serve_quickwit(config, async move {
```

Comment:
I think we should take into account other signals (I had this [kind of stuff in mind](https://github.com/vectordotdev/vector/blob/68b54a9bc0ae07d916ec48e997a03f7681e54ccc/src/signal.rs#L44)
I suggest to do that in another PR so that you can advance on your original issue :)

Reply:
There is an issue related to this https://github.com/quickwit-oss/quickwit/issues/3073